core/handler/cowboy_routing.ex

```diff
@@ -28,7 +30,24 @@ defmodule AntikytheraCore.Handler.CowboyRouting do
       @total_error_count_route,
       @per_app_error_count_route,
     ]
-    [{:_, path_rules}]
+    [{:_, default_gear_routes() ++ path_rules}]
```

Comment:
I understand that this is WIP.
What happens if a gear has `/healtheck` path?
I don't think it should work, but we need some documentation about it.

Reply:
`/healthcheck` returns the gear API instead of antikythera.
I think this behavior is ok because gear developers don't use antikythera's health check API.

Or do you mean the following implementation would be better?
It may be confusing for gear developers that APIs not written in `router.ex` return responses.
```elixir
[{:_, default_gear_routes()]
```

Reply:
> /healthcheck returns the gear API instead of antikythera.
> I think this behavior is ok because gear developers don't use antikythera's health check API.

SGTM.
I don't think we need a documentation if gear API overwrites Antikythera API